versuch1/src/main/java/Receive_Mail.java

```diff
@@ -12,6 +12,31 @@ public static void main(String[] args) throws Exception {
 	public static void fetchMail() {
 		try {
 			// your code here
+			Properties properties = new Properties();
+			properties.put("mail.pop3.host", "localhost");
+			properties.put("mail.pop3.port", "110");
+
+			Session session = Session.getDefaultInstance(properties, null);
+
+			Store store = session.getStore("pop3");
+			store.connect("localhost", "labrat", "kn1lab");
+
+			Folder folder = store.getFolder("INBOX");
+			folder.open(Folder.READ_ONLY);
+
+			Message[] messages = folder.getMessages();
+			System.out.println("Messages:");
+			System.out.println(messages.length);
+			for (int i = 0, n = messages.length; i < n; i++){
```

Comment:
If messages length is 0, also keine Nachrichten, dann soll man villeicht zeigen, das kein Messages gibt.
z.B.: "Message folder is empty"